collector/collectors.go

```diff
@@ -26,8 +29,17 @@ func NewFlashbladeCollector(fbClient *fb.FlashbladeClient) *FlashbladeCollector
 	arrayPerformanceCollector := NewArrayPerformanceCollector(fbClient)
 	bladesCollector := NewBladesCollector(fbClient)
 	filesystemsCollector := NewFilesystemsCollector(fbClient)
-
-	subcollectors := []Subcollector{alertsCollector, arrayPerformanceCollector, bladesCollector, filesystemsCollector}
+	usageCollector := NewUsageCollector(fbClient)
+	fsPerformanceCollector := NewFSPerformanceCollector(fbClient)
```

Comment:
The two new collectors are awesome! Thanks for adding them.

We may have to make these collectors optional with a command line flag, because they make API calls per filesystem (of which you can have quite a few on a FlashBlade, potentially thousands). Perhaps enable these collectors with `--filesystem-metrics`? What do you think?

Reply:
That's a good point. I'll add that that flag.